File: Melobit_player_java/app/src/main/java/melo_beat/models/Query/Song.java
package melo_beat.models.Query;

import java.util.List;

public class Song{
	private int duration;
	private boolean hasVideo;
	private boolean copyrighted;
	private Image image;
	private List<ArtistsItem> artists;
	private String releaseDate;
	private Album album;
	private boolean localized;
	private String id;
	private Audio audio;
	private String title;
	private String downloadCount;

	public int getDuration(){
		return duration;
	}

	public boolean isHasVideo(){
		return hasVideo;
	}

	public boolean isCopyrighted(){
		return copyrighted;
	}

	public Image getImage(){
		return image;
	}

	public List<ArtistsItem> getArtists(){
		return artists;
	}

	public String getReleaseDate(){
		return releaseDate;
	}

	public Album getAlbum(){
		return album;
	}

	public boolean isLocalized(){
		return localized;
	}

	public String getId(){
		return id;
	}

	public Audio getAudio(){
		return audio;
	}

	public String getTitle(){
		return title;
	}

	public String getDownloadCount(){
		return downloadCount;
	}
}

File: tina/app/src/main/java/com/example/musicplayer/models/searchQuery/Song.kt
package com.example.musicplayer.models.searchQuery

data class Song(
    val album: AlbumX,
    val artists: List<ArtistXXX>,
    val audio: Audio,
    val copyrighted: Boolean,
    val downloadCount: String,
    val duration: Int,
    val hasVideo: Boolean,
    val id: String,
    val image: ImageXXXXXX,
    val localized: Boolean,
    val releaseDate: String,
    val title: String
)